airctrl/airctrl.py

```diff
@@ -87,6 +87,11 @@ def load_key(self):
                 self._get_key()
         else:
             self._get_key()
+        self._check_key()
```

Comment:
_get_key() always provides a valid key, so you can move this check after line 85